ui/pages/swaps/prepare-swap-page/not-enough-balance.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  TextVariant,
+  Size,
+  Severity,
+} from '../../../helpers/constants/design-system';
+import {
+  BannerAlert,
+  ButtonLink,
+  Text,
+} from '../../../components/component-library';
+
+interface Props {
+  title: string;
+  actionableBalanceErrorMessage: string;
+  needsMoreGas: boolean;
+  openBuyCryptoInPdapp: () => void;
+  needsMoreGasText: string;
+}
+
+export default function NotEnoughBalance({
+  title,
+  actionableBalanceErrorMessage,
+  needsMoreGas,
+  openBuyCryptoInPdapp,
+  needsMoreGasText,
+}: Props) {
+  return (
+    <BannerAlert
+      severity={Severity.Info}
+      title={title}
+      className="not-enough-balance__banner"
```

Comment:
Thanks for adding it here! Just one small nit to remove the unused CSS with this file and it will be good to go.

Reply:
@NidhiKJha TS will complain if the `className` is removed.

```
Property 'className' is missing in type '{ children: Element; severity: Severity; title: string; }' but required in type '{ [x: string]: any; children: any; className: any; severity?: string | undefined; }'.ts(2741)
```

Please let me know what you like me to do

Reply:
This is exactly what I was referring to. To add a className but no CSS as we don't need styling. Thanks, @infiniteflower